script.js

```diff
@@ -29,3 +29,148 @@ menuItem3.addEventListener("click", () => {
   menu.classList.remove("menu");
   menu.classList.add("hidden");
 });
+
+const portfolio = document.querySelector('#portfolio');
+
+const body = document.body;
+  const article = document.createElement('article');
+    const div0 = document.createElement('div');
+      const div1 = document.createElement('div');
+        const div2 = document.createElement('div');
+          const h3 = document.createElement('h3');
+          const i0 = document.createElement('i');
+        const img0 = document.createElement('img');
+        const p0 = document.createElement('p');
+      const ul = document.createElement('ul');
+        const li0 = document.createElement('li');
+        const li1 = document.createElement('li');
+        const li2 = document.createElement('li');
+        const li3 = document.createElement('li');
+      const button0 = document.createElement('button');
+        const p1 = document.createElement('p');
+        const a1 = document.createElement('a');
+          const i1 = document.createElement('i');
+      const button1 = document.createElement('button');
+        const p2 = document.createElement('p');
+        const a2 = document.createElement('a');
+          const img1 = document.createElement('img');
+
+
+
+let names = ['Multi-Post Stories', 'Multi-Post Stories', 'Multi-Post Stories', 'Multi-Post Stories'];
```

Comment:
- [ ] I think the close button is missing from the pop-up window. Please add it.
- [ ] I can hardly read the project description and the technology. Please adjust the opacity or transparency so that we can see clearly.
- [ ] The button `see live` and `see source` is aligned vertically in your design whereas in figma design it is aligned horizontally in `desktop` version.

### Your design

![Screenshot (15)](https://user-images.githubusercontent.com/22434660/127540143-8526c859-975e-4fa5-83ea-4ba8ce334c72.png)

### Figma design

![Screenshot (16)](https://user-images.githubusercontent.com/22434660/127540534-bfe7f8f7-6f43-4a39-8c77-7d9e7fe60fd7.png)